### PR TITLE
Needed changes following the manageiq api client rename

### PR DESCRIPTION
**Pull request by dkorn**

### CODE REVIEW

**Comment by dkorn** [comment]
@cben @enoodle @jaryn PTAL

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
--e git+https://github.com/ManageIQ/manageiq-api-client-python.git#egg=manageiq-api-client-python
--e .
+manageiq-client
```
Comment (enoodle) [inline requirements.txt:1]: :+1:

**Review by enoodle — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Ansible modules which provide different ManageIQ operations.
 
 ManageIQ Python API Client package [manageiq-api-client-python] (https://github.com/ManageIQ/manageiq-api-client-python/).
 
-    `$ pip install git+https://github.com/ManageIQ/manageiq-api-client-python.git`
+    `$ pip install manageiq-client`
```
Comment (cben) [inline README.md:14]: consider saying `$ pip install -r requirements.txt`

also line above should probably be `[manageiq-client](...)`

**Review by cben — APPROVED** [review]
Let's Make Travis Green Again

File: README.md (hunk 2 above)
Comment (dkorn) [reply README.md:14]: I left line 14 as is cause this part only refers only to client and the `requirements.txt` may also include other packages.

changed `[manageiq-api-client-python]` to `[manageiq-client]`

**Pull request merged**